soccer/src/soccer/strategy/agent/position/pivot_test.hpp

```diff
@@ -0,0 +1,56 @@
+#pragma once
+
+#include <chrono>
+#include <cmath>
+#include <string>
+#include <unordered_map>
+
+#include <rclcpp/rclcpp.hpp>
+#include <rclcpp_action/rclcpp_action.hpp>
+#include <spdlog/spdlog.h>
+
+#include <rj_msgs/action/robot_move.hpp>
+
+#include "planning/instant.hpp"
+#include "position.hpp"
+#include "rj_common/time.hpp"
+#include "rj_constants/constants.hpp"
+#include "rj_geometry/geometry_conversions.hpp"
+#include "rj_geometry/point.hpp"
+#include "seeker.hpp"
+
+namespace strategy {
+
+class Pivot : public Position {
```

Comment:
note the clang-tidy warning you need to define some special member functions  see "rule of five" here https://en.cppreference.com/w/cpp/language/rule_of_three

Reply:
note that you can explicitly `= default;` them